### DISCUSSION TITLE
Is there a way to exclude a field from a `find` operation

### DISCUSSION THREAD

**Discussion by kujtimiihoxha** [sensedeep/dynamodb-onetable#249] [Q&A] [answered]
I have some data that I want to exclude when doing `find` operations to return less data and make these queries faster, am I missing a feature or is this not implemented.

Thanks.

**Comment by mobsense** [comment, marked as answer]
Set parma.fields to the attributes you want to return.

| fields | `array` | List of properties to return. This sets the ProjectionExpression. Default null. |

**Reply by kujtimiihoxha** [reply]
Will this set the [AttributesToGet](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/LegacyConditionalParameters.AttributesToGet.html) property so that it omits them from the DB request also not only when they are mapped ?

**Reply by mobsense** [reply]
AttributesToGet is legacy. ProjectionExpression is the replacement. We use that. And yes, only those attributes are returned from DDB.

**Reply by kujtimiihoxha** [reply]
Awesome thanks 👍
